docs/website/mmp-errors.md

```diff
@@ -104,6 +108,17 @@ A last-straw solution would be to use an older version of Xamarin.Mac, one that
 
 ### <a name="MM0114">MM0114: Hybrid AOT compilation requires all assemblies to be AOT compiled.
 
+### <a name="MT0129"/>MT0129: Debugging symbol file for '*' does not match the assembly and is ignored.
```

Comment:
Should be MM, not MT

Reply:
oops, missed that one (but somehow did the MM0009)